Anguandia/quickcredit
# Get specific loan from database

#### Purpose of this pull request

Integrate database in get specific loan feature

#### Tasks to be accomplished

- Add database query capability for fetching target loan to
loanController/detail function
- Add Capability to detect request and generate an appropriate response for
missing target in loanNot function in availability check utility
function
- Add loanNot target check function as a middleware to the
GET/loans/:loanId route for getting a specific loan

#### How to manually test this feature

- From your local copy of the app, populate the database with some loans
- Restart the app and request a known loan with a GET/loans/<loanId>
request on /api/v1 using localhost and port 3000 on postman
- The response should return the requested loan's details
- Now request for a loan with a non-existing Id,
- The response should inform of non-existence

#### Affected pivotal tracker stories

[finishes #166115135 #166114379]
